src/main/java/nl/tudelft/ti2206/group9/gui/GameScene.java

```diff
@@ -22,7 +23,6 @@
 import nl.tudelft.ti2206.group9.util.SaveGame;
 
 import static nl.tudelft.ti2206.group9.ShaftEscape.OBSERVABLE;
```

Comment:
This import is twice in this file. Please remove one of them.

<organization,review